extensions/adapters/vector/src/main/java/mil/nga/giat/geowave/adapter/vector/plugin/GeoWaveFeatureReader.java

```diff
@@ -260,13 +263,17 @@ public BaseIssuer(
 		public CloseableIterator<SimpleFeature> query(
 				final PrimaryIndex index,
 				final mil.nga.giat.geowave.core.store.query.Query query ) {
+			final QueryOptions queryOptions = new QueryOptions(
```

Comment:
Certainly could, though I'm personally of the opinion it's better to have constructors covering the required fields and setters for optional fields vs. a ton of permutations of constructors.